### PR TITLE
Made connections to test the connection

### PR DESCRIPTION
**Pull request by wooseok0717**
- set up airbnb linter
- created connection to db (although it is not connected to models)
- set up routers to MVC style.

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -13,8 +14,10 @@ app.use(morgan('dev'));
 app.use(cors());
 app.use(express.json());
 
+app.use('/api', router);
```
Comment (donnatopia) [inline server/index.js]: it is probably better to use the relative path "/reviews" to be more specific.

File: server/db/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const { Client } = require('pg');
+
+const client = new Client({
```
Comment (donnatopia) [inline server/db/index.js:3]: I am not familiar with Client, but this looks so clean

File: server/db/index.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  host: 'localhost',
```
Comment (donnatopia) [inline server/db/index.js:4]: You should pull these variables from dotenv instead of hardcoding

File: server/controllers/index.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+const models = require('../models');
+
+module.exports = {
+  sayHello: (req, res) => {
+    const something = models.sayHello();
+    res.send(something);
```
Comment (donnatopia) [inline server/controllers/index.js:7]: don't forget to send status codes!

**Review by donnatopia — APPROVED** [review]
Small changes for when you actually implement model and controller methods for your reviews api. Otherwise, good job!

File: server/db/index.js (hunk 3 above)
Comment (wooseok0717) [reply server/db/index.js:4]: changed this! thank you!

File: server/db/index.js (hunk 2 above)
Comment (wooseok0717) [reply server/db/index.js:3]: ❤️

File: server/index.js (hunk 1 above)
Comment (wooseok0717) [reply server/index.js]: change done!

**Review by donnatopia — APPROVED** [review]

**Comment by donnatopia** [comment]
nice changes! go ahead and merge

**Pull request merged**